main.cpp

```diff
@@ -13,75 +13,104 @@ What you create in this project will be used as the basis of Project 5 in the co
-    Name 3 things you'll find on the:    Infotainment System
-        1) radio channel controls
-        2) sat nav controls
-        3) system volume controls
-    Name 3 things you can do with the:   Infotainment System
-        1) change radio station
-        2) set sat nav
-        3) change system volume levels
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1) height adjustment controls
-        2) tilt adjustment controls
-        3) position controls
-    Name 2 things you can do with the:   Seat
-        1) adjust seat height
-        2) adjust seat position
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - the amount of water used per week.
+        - amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) Hairdressing Salon
+5 properties:
+    1) Number of Hairdressers
+    2) Number of Colourists
+    3) Number of Units of Shampoo
+    4) Number of Units of Hair Colour
+    5) Number of Customers
+3 things it can do:
+    1) Cut Hair
+    2) Colour Hair
+    3) Wash Hair
+ */
+
+/*
+Thing 2) Audio Compressor Limiter Effect
+5 properties:
+    1) Threshold for Compressor
+    2) Make Up Gain for Compressor
+    3) Threshold for Limiter
+    4) Amount of Gain Change
+    5) Compression ratio
+3 things it can do:
+    1) Compress Audio
+    2) Limit Audio 
+    3) Change Audio Gain
+ */
+
+/*
+Thing 3) Gym
+5 properties:
+    1) Annual Fee
+    2) Number of Customers
+    3) Number of Running Machines
+    4) Staff Wages
+    5) Annual Income
+3 things it can do:
+    1) Pay Staff
+    2) Service Machines
+    3) Invoice Customers
+ */
+
+/*
+Thing 4) Fridge Freezer
+5 properties:
+    1) Temperature of Fridge
+    2) Temperature of Freezer
+    3) Available Space in Fridge
+    4) Available Space in Freezer
+    5) 
+3 things it can do:
+    1) Change Fridge Temperature
+    2) Change Freezer Temperature
+    3) Check Temperatures
```

Comment:
You are missing a property.
Feel free to change `Thing 4)` to a different object if it'll be easier to come up with 5 properties.
- [ ] revise

Reply:
I have amended Thing 4)